query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/persist/CreateLdapConfiguration.java

```diff
@@ -146,6 +146,7 @@ public BooleanField performFunction() {
 
   @Override
   public void validate() {
+    config.connectionField().isRequired(true);
```

Comment:
❗️ This should be marked as required where all the other fields are being marked required in the constructor.
If this line is added back https://github.com/connexta/admin-console/pull/212/files#diff-6f7bb971eeb8ea931b9594698af3b5f1L139 then you won't have  to marked this as required at all since it will be covered by the `useDefaultRequired` method https://github.com/connexta/admin-console/pull/212/files#diff-07a3621f580f3c35ab6403b229ae48c2R69